mobile-app/cypress/integration/functional/wallet/balances/balances.spec.ts

```diff
@@ -6,6 +8,13 @@ export interface BalanceTokenDetail {
   usdAmount?: string
 }
 
+function checkValueWithinRange (actualVal: string, expectedVal: string): void {
+  const value = new BigNumber(actualVal.replace('$', '').replace(',', ''))
+  const expectedValue = new BigNumber(expectedVal)
+  expect(value.gte(expectedValue.minus(2))).to.be.eq(true)
```

Comment:
question: why we choose `2` here, any specific reason?

Reply:
@kyleleow I think he added it to handle rewards. Rewards are generated fast in playground so hard to get actual value